src/main/java/ca/warp7/frc2022/commands/TeleopCommand.java

```diff
@@ -136,57 +150,33 @@ else if (driver.rightBumper.isReleased())
             setLowGearDriveCommand.schedule();
 
         if (!isIntaking) {
-            isIntaking = driver.leftTrigger > 0.22;
-        } else {
-            isIntaking = driver.leftTrigger > 0.2;
-        }
-
-        if (!isFeeding) {
-            isFeeding = driver.rightTrigger > 0.22;
+            isIntaking = driver.rightTrigger > 0.22;
         } else {
-            isFeeding = driver.rightTrigger > 0.2;
+            isIntaking = driver.rightTrigger > 0.2;
         }
 
         if (driver.yButton.isPressed()) {
             isReversed = !isReversed;
         }
 
-        if (driver.aButton.isPressed()) {
-            visionAlignCommand.schedule();
-        } else if (driver.aButton.isReleased()) {
-            visionAlignCommand.cancel();
-            curvatureDriveCommand.schedule();
-        }
-
         // Operator
 
-        if (operator.leftTrigger > 0.2) {
-            isPriming = true;
-            isClose = true;
-            Limelight.getInstance().setPipeline(0);
-        } else if (operator.rightTrigger > 0.3) {
-            isPriming = true;
-            isClose = false;
-            Limelight.getInstance().setPipeline(1);
-        } else
-            isPriming = false;
-
-        if (operator.leftBumper.isPressed()) {
-            if (isClose) closeShotAdjustment -= 0.5;
-            else farShotAdjustment -= 0.5;
+        if (operator.yButton.isPressed()) {
+            isHighGoal = !isHighGoal;
         }
+
         if (operator.rightBumper.isPressed()) {
-            if (isClose) closeShotAdjustment += 0.5;
-            else farShotAdjustment += 0.5;
+            isLaunching = !isLaunching;
         }
-        if (operator.xButton.isPressed()) {
-            if (isClose) closeShotAdjustment -= 5;
-            else farShotAdjustment -= 5;
-        }
-        if (operator.yButton.isPressed()) {
-            if (isClose) closeShotAdjustment += 5;
-            else farShotAdjustment += 5;
+
+        if (!isFeeding) {
+            isFeeding = operator.rightTrigger > 0.22;
+        } else {
+            isFeeding = operator.rightTrigger > 0.2;
```

Comment:
One comment here.  These values are close together, and I think this is an analog input on the controller.  Isn't this almost...

isFeeding = operator.rightTrigger > .2;

i.e.,  no matter what, change the next isFeeding state to the operator's rightTrigger input as long as it's over some minimum threshold (.2).